doc/reference/peripherals/uart.rst

```diff
@@ -6,7 +6,69 @@ UART
 Overview
 ********
 
+Zephyr provides three different ways to access the UART peripheral. Depending
```

Comment:
maybe it would be worth to point out that these are 3 APIs for transferring data over UART. Apart from that there is API for controlling/configure UART which is common. It could be also grouped together into another doxygen group as well as in header file. Currently, they are randomly placed in the header file.

Reply:
In the documentation all common functions (not assigned to one of the groups) are automatically placed before the three newly introduced groups. Do you think that would be sufficient? Unfortunately I don't have an idea how I could quickly show the rendered version...

Reply:
The rendered versions can be downloaded from the checks page on this PR (e.g. https://github.com/zephyrproject-rtos/zephyr/actions/runs/1658802622).